### PR TITLE
first branch commit

### PR DESCRIPTION
**Pull request by eddggs**
Please approve

### CODE REVIEW

**Review by wangx6 — APPROVED** [review]
good work

File: homework2.html [hunk 1]
```diff
@@ -296,6 +296,7 @@
     
         </div>
     </div>
+    <div>adding something</div>
```
Comment (merceearagonees) [inline homework2.html]: dgrgfhgfg

File: leson1 27Feb.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
     margin-bottom: 30px;
 }
 .app-submit{
-    background-color: black;
+    background-color: red;
     color: white;
```
Comment (merceearagonees) [inline leson1 27Feb.html:15]: grgrgg

**Review by merceearagonees — CHANGES_REQUESTED** [review]

File: homework2.html (hunk 1 above)
Comment (eddggs) [reply homework2.html]: I am so sorry

File: leson1 27Feb.html (hunk 2 above)
Comment (eddggs) [reply leson1 27Feb.html:15]: I will fix it

**Review by wangx6 — APPROVED** [review]

**Review by merceearagonees — APPROVED** [review]

**Pull request merged**